build.assets/windows/build.ps1

```diff
@@ -354,25 +361,80 @@ function Build-Tsh {
         $UnsignedBinaryPath = "$BuildDirectory\unsigned-$BinaryName"
         go build -tags piv -trimpath -ldflags "-s -w" -o "$UnsignedBinaryPath" "$TeleportSourceDirectory\tool\tsh"
         if ($LastExitCode -ne 0) {
-           exit $LastExitCode
+            exit $LastExitCode
         }
         Write-Host "::endgroup::"
 
         Write-Host "::group::Signing tsh..."
         Invoke-SignBinary -UnsignedBinaryPath "$UnsignedBinaryPath" -SignedBinaryPath "$SignedBinaryPath"
         Write-Host "::endgroup::"
+    }
+    Write-Host $("Built TSH in {0:g}" -f $CommandDuration)
+
+    return "$SignedBinaryPath"  # This is needed for building Connect and bundling the zip archive
+}
+
+function Build-Tctl {
```

Comment:
`tctl.exe` only exists for v16 and up.

Reply:
Thanks. I think I did `git checkout commit -- file_path` instead of `git cherry-pick` which is why this was included. I'll get it fixed.